README.md

```diff
@@ -954,42 +954,72 @@ Used for EV chargers.
-Type | Interface                      | Value type | Properties              | Description
------|--------------------------------|------------|-------------------------|------------
-in   | cmd.charge.start               | null       |                         | Start charging (allow cars to charge) OCPP: Remote Start Transaction
-in   | cmd.charge.stop                | null       |                         | Stop charging (stop cars from charging) OCPP: Remote Stop Transaction
+Type | Interface                       | Value type | Properties              | Description
+-----|---------------------------------|----------|-------------------------|------------
+in   | cmd.charge.start                | null     |                         | Start charging (allow cars to charge) OCPP: Remote Start Transaction
+in   | cmd.charge.stop                 | null     |                         | Stop charging (stop cars from charging) OCPP: Remote Stop Transaction
 -|||
-in   | cmd.state.get_report           | null       |                         | Get the state of the chargepoint, see sup_states
-out  | evt.state.report               | string     |                         | State report of the chargepoint
+in   | cmd.state.get_report            | null     |                         | Get the state of the chargepoint, see sup_states
+out  | evt.state.report                | string   |                         | State report of the chargepoint
 -|||
-in   | cmd.cable_lock.set             | bool       |                         | Lock & unlock the cable/connector
-in   | cmd.cable\_lock.get\_report    | null       |                         | Get the status of the cable_lock
-out  | evt.cable_lock.report          | bool       |                         | true = locked, false = unlocked
+in   | cmd.cable_lock.set              | bool     |                         | Lock & unlock the cable/connector
+in   | cmd.cable\_lock.get\_report     | null     |                         | Get the status of the cable_lock
+out  | evt.cable_lock.report           | bool     |                         | true = locked, false = unlocked
 -|||
-in   | cmd.access.set                 | string     |                         | 
-in   | cmd.access.get_report          | null       |                         |
-out  | evt.access.report              | string     |                         |
+in   | cmd.access.set                  | string   |                         | 
+in   | cmd.access.get_report           | null     |                         |
+out  | evt.access.report               | string   |                         |
 -|||
-in   | cmd.smart_charge.set           | bool       |                         | true = enable smart charging
-in   | cmd.smart_charge.get_report    | null       |                         | 
-out  | evt.smart_charge.report        | bool       |                         |
+in   | cmd.smart_charge.set            | bool     |                         | true = enable smart charging
```

Comment:
Please sync with Markus and remove smart_charge and rated_current if it is not used by any of our adapters (which I think it is not). We pollute the documentation with features that may never be supported, let's clean this up.

Reply:
Markus confirmed it's not used anywhere, I'll remove it...

Reply:
Done.